docs/changehistory/NextVersion.md

```diff
@@ -78,19 +78,30 @@ It was, however, confusing from a usage and documentation perspective and needed
 ## Promoted APIs
 
 The following APIs have been promoted to `public`. Public APIs are guaranteed to remain stable for the duration of the current major version of a package.
+
 ### [@bentley/bentleyjs-core](https://www.itwinjs.org/reference/bentleyjs-core/)
-  * [assert]($bentleyjs-core) for asserting logic invariants.
-  * [ProcessDetector]($bentleyjs-core) for querying the type of executing Javascript process.
-  * [ObservableSet]($bentleyjs-core) for a [Set](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set) that emits events when its contents are modified.
-  * [ByteStream]($bentleyjs-core) for extracting data from binary streams.
-  * Types related to collections of [Id64String]($bentleyjs-core)s
-    * [OrderedId64Iterable]($bentleyjs-core) and [OrderedId64Array]($bentleyjs-core)
-    * [CompressedId64Set]($bentleyjs-core) and [MutableCompressedId64Set]($bentleyjs-core)
+
+* [assert]($bentleyjs-core) for asserting logic invariants.
+* [ProcessDetector]($bentleyjs-core) for querying the type of executing JavaScript process.
+* [ObservableSet]($bentleyjs-core) for a [Set](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set) that emits events when its contents are modified.
+* [ByteStream]($bentleyjs-core) for extracting data from binary streams.
+* Types related to collections of [Id64String]($bentleyjs-core)s
+  * [OrderedId64Iterable]($bentleyjs-core) and [OrderedId64Array]($bentleyjs-core)
+  * [CompressedId64Set]($bentleyjs-core) and [MutableCompressedId64Set]($bentleyjs-core)
 
 ### [@bentley/hypermodeling-frontend](https://www.itwinjs.org/reference/hypermodeling-frontend/)
 
 All hyper-modeling APIs are now public. [This interactive sample](https://www.itwinjs.org/sample-showcase/?group=Viewer&sample=hypermodeling-sample&imodel=House+Sample) demonstrates how to use hyper-modeling features.
 
+### [@bentley/imodeljs-common](https://www.itwinjs.org/reference/imodeljs-common/)
+
+* [ThematicDisplay]($frontend) for colorizing a [Viewport]($frontend)'s scene based on aspects of the rendered geometry. [This interactive sample](https://www.itwinjs.org/sample-showcase/?group=Viewer+Features&sample=thematic-display-sample&imodel=CoffsHarborDemo) demonstrates the usage of thematic display.
```

Comment:
ThematicDisplay object is in common - should this be under the heading for imodeljs-common?

Reply:
@pmconne I must have been looking at an older code version.  This is actually now under the imodeljs-common heading as expected.  However, should  ThematicDisplay be tagged ($common) instead of ($frontend)?

Reply:
Please fix.

Reply:
fixed.